File: DAM2/DDI/Ejercicios/Tema2/Ficheros/libros.vb
Public Class libros
    Private titulo, autor As String
    Private isbn, cantidad As Integer

    Public Sub New(titulo As String, autor As String, isbn As Integer, cantidad As Integer)
        Me.titulo = titulo
        Me.autor = autor
        Me.isbn = isbn
        Me.cantidad = cantidad
    End Sub

    Public Sub setTitulo(titulo As String)
        Me.titulo = titulo
    End Sub

    Function getTitulo() As String
        Return Me.titulo
    End Function

    Public Sub setAutor(autor As String)
        Me.autor = autor
    End Sub

    Function getAutor() As String
        Return Me.autor
    End Function

    Public Sub setIsbn(isbn As String)
        Me.isbn = isbn
    End Sub

    Function getIsbn() As Integer
        Return Me.isbn
    End Function

    Function getCantidad() As Integer
        Return Me.cantidad
    End Function
End Class

File: DAM2/AAD/Ejercicios Resueltos/GarciaSanchez_JuanRoberto_2EVA_Ficheros/src/Libros.java

import java.io.Serializable;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Juanro49
 */
public class Libros implements Serializable
{
    int codigo, fecha_edicion;
    String titulo, autor, editorial, isbn;
    
    public Libros(int codigo, String titulo, String autor, String editorial, int fecha_edicion, String isbn)
    {
        setCodigo(codigo);
        setTitulo(titulo);
        setAutor(autor);
        setEditorial(editorial);
        setFecha_edicion(fecha_edicion);
        setIsbn(isbn);
    }

    public int getCodigo()
    {
        return codigo;
    }

    public void setCodigo(int codigo)
    {
        this.codigo = codigo;
    }

    public int getFecha_edicion()
    {
        return fecha_edicion;
    }

    public void setFecha_edicion(int fecha_edicion)
    {
        this.fecha_edicion = fecha_edicion;
    }

    public String getTitulo()
    {
        return titulo;
    }

    public void setTitulo(String titulo)
    {
        this.titulo = titulo;
    }

    public String getAutor()
    {
        return autor;
    }

    public void setAutor(String autor)
    {
        this.autor = autor;
    }

    public String getEditorial()
    {
        return editorial;
    }

    public void setEditorial(String editorial)
    {
        this.editorial = editorial;
    }

    public String getIsbn()
    {
        return isbn;
    }

    public void setIsbn(String isbn)
    {
        this.isbn = isbn;
    }
    
    
}
